<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SiteEditor</class>
 <widget class="QWidget" name="SiteEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Web App</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="3">
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="iconStatusLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="iconLocationLineEdit"/>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="urlLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>URL is opened when the web app is launched. Links which do not begin with this URL are opened in a web browser.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="iconPreviewLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="checkBoxBrowserPlugins">
       <property name="text">
        <string>Enable browser plugins</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="newButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Discard|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="existingButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>urlLineEdit</tabstop>
  <tabstop>checkBoxBrowserPlugins</tabstop>
  <tabstop>iconLocationLineEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>newButtonBox</tabstop>
  <tabstop>existingButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
